File: descendingOrder.js
function descendingOrder(n){
  if (n.toString().length > 1) {
    let sortedNums = n.toString().split('').sort((a, b) => b - a).join('');
    return parseInt(sortedNums);
  } else {
    return n;
  }
}

function descendingOrder(n){
  return (n.toString().length > 1) ? parseInt(n.toString().split('').sort((a, b) => b - a).join('')) : n;
}

function descendingOrder(n){
  return parseInt(n.toString().split('').sort((a, b) => (b - a)).join(''));
}

descendingOrder(123456789);

File: descending_order.rb
def descending_order(n)
   return n.to_s.split('').sort.reverse.join('').to_i
end

# sort doesn't need parentheses
def descending_order(n)
  n.digits.sort().reverse.join.to_i
end

def descending_order(n)
  n.digits.sort.reverse.join.to_i
end

descending_order(0)
